packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -218,4 +222,16 @@ private void registerJavaScriptChannelNames(List<String> channelNames) {
   public void dispose() {
     methodChannel.setMethodCallHandler(null);
   }
+
+  private void enableGeolocation() {
+    webView.getSettings().setGeolocationEnabled(true);
+    webView.setWebChromeClient(
+        new WebChromeClient() {
+          @Override
+          public void onGeolocationPermissionsShowPrompt(
+              String origin, GeolocationPermissions.Callback callback) {
+            callback.invoke(origin, true, false);
```

Comment:
We shouldn't do that, the developer should control which sites can access geolocation and which sites cannot.